src/platform_impl/web/event_loop/window_target.rs

```diff
@@ -168,20 +168,22 @@ impl<T> WindowTarget<T> {
 
         let runner = self.runner.clone();
         let raw = canvas.raw().clone();
-        let mut intended_size = LogicalSize {
-            width: raw.width() as f64,
-            height: raw.height() as f64,
+        // todo
+        let mut intended_size = PhysicalSize {
```

Comment:
The canvas' width and height are measured in physical units (actual pixels). This is scaled down (or up) via the css size